main/src/ca/uwaterloo/flix/tools/pkg/IncludedModules.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Holger Dal Mogensen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.uwaterloo.flix.tools.pkg
+
+import ca.uwaterloo.flix.language.ast.Symbol
+
+sealed trait IncludedModules {
+  def contains(sym: Symbol.ModuleSym): Boolean
```

Comment:
The implementation should be here, by pattern matching on this.

We dont use the OO style with override :)